ddtrace/tracer.py

```diff
@@ -551,7 +551,6 @@ def _start_span(
             )
             span._local_root = span
             span.metrics[system.PID] = self._pid or getpid()
-            span.meta["runtime-id"] = get_runtime_id()
```

Comment:
Do we not need to set `runtime-id` on a span if it's not a distributed part of a distributed trace?

Reply:
@Kyle-Verhoog looking at this, do we need to do the same for `system.PID` as well?

Reply:
We do still need to @Yun-Kim. In general we want root spans to have the runtime-id and os pid.

Reply:
I thought in distributed tracing the new root span technically should still have a parent id right? Or is there not a strong parent reference in a distributed tracing context?

Reply:
Correct, it has a parent id and not a strong parent reference.

Using `_parent` isn't _great_ but it's the best we can do I think. It does mean that we'll report the runtime id and os pid for intra-process execution crossing in the local root span (like the local root of a new thread).